### ISSUE TITLE
[BUG] Persist commands have inconsistent behaviour in the Properties section.

### ISSUE THREAD

**Issue by irm-codebase** [mgmeyers/obsidian-zotero-integration#317]
Hello!

Here is my use-case: I subdivide note metadata into Zotero-specific stuff (zotero tags, title, authors, etc.), and Obsidian stuff (obsidian tags, extra attributes that you can only get by understanding the article like region, economic sector, etc.).

This creates a problem: I may want to re-run a template, updating the Zotero metadata, and keep my Obsidian notes and metadata intact.

The persist command works well enough on the notes side, but it breaks stuff if you put it on the metadata. I managed to "cheat" the problem by just completely ignoring the metadata:

```
---
{% persist "meta" %}
aliases: ["{{title}}"]
authors: [{{authors}}]
{% if journalAbbreviation %}journal: "{{journalAbbreviation}}"{% endif %}
{% if bookTitle %}book: "{{bookTitle}}"{% endif %}
year: {{date | format ("YYYY")}}
{% if DOI %}DOI: https://doi.org/{{DOI}}{% endif %}
zotero tags: [{{allTags}}]
zotero link: zotero://select/items/@{{citekey}}
zotero type: "{{itemType}}" 
obsidian type: "zotero note"
model:
region:
tags: 
date: {{importDate | format("YYYY-MM-DD")}} {% endpersist %}
---
```

But if you move `{% persist "meta" %}` or `{% endpersist %}` **anywhere else** the code will break. I believe this is an Obsidian-side issue, but a workaround would be nice since it might be ignored by the developers.

**Comment by irm-codebase** [comment]
Actually, the issue is much, much worse:
Obsidian seems to actively delete any comment text in the properties section if you add a new property or modify it.

Basically, the persistence of the metadata cannot be assured, as of now.

**Comment by FeralFlora** [comment]
> Basically, the persistence of any modifications to the metadata cannot be assured, as of now.

If you use Dataview, I recommend a workaround that entails storing metadata you want to persist in a persistent field in the note content as Dataview inline fields.

**Comment by notuntoward** [comment]
Agreed.  There is a serious need for a way to either prevent literature note overwrite (#351 #281 #201 ), or at a minimum, a way to persist properties (#157 #138)
